Testing/integration/allow_sysex.scpt

```diff
@@ -0,0 +1,30 @@
+on run argv
```

Comment:
Do we need this if `systemextensionsctl developer on` is set?

Reply:
It also affects the upgrades - allowing upgrades to/from any version number